### PR TITLE
feat(nutanix): add contact management to nutanix

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nutanix`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7504
| License          | BSD 3-Clause

## Description

add contact management tile on nutanix service

### CODE REVIEW

File: packages/manager/modules/nutanix/package.json [hunk 1]
```diff
@@ -12,16 +12,19 @@
   "author": "OVH SAS",
   "main": "./src/index.js",
   "dependencies": {
+    "@ovh-ux/manager-models": "^1.4.3",
     "bootstrap4": "twbs/bootstrap#v4.0.0",
     "lodash": "^4.17.14"
   },
   "peerDependencies": {
+    "@ovh-ux/manager-billing-components": "^2.0.0 || ^3.0.0",
```
Comment (antleblanc) [inline packages/manager/modules/nutanix/package.json:20]: Following module has `moment` has peerDependency which means that the standalone
application `@ovh-ux/manager-nutanix-app` must have it as a `dependency`.

File: packages/manager/modules/nutanix/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "globals": {
+    "moment": true
```
Comment (antleblanc) [inline packages/manager/modules/nutanix/.eslintrc.json:3]: **question:** Needed?

Can't find into following location:
```sh
$ grep -rnw 'packages/manager/modules/nutanix/src' -e 'moment' | wc -l
0
```

File: packages/manager/apps/nutanix/.eslintrc.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "globals": {
     "__VERSION__": true,
+    "moment": true,
```
Comment (antleblanc) [inline packages/manager/apps/nutanix/.eslintrc.json:4]: question: Needed?

Can't find into following location:
```sh
$ grep -rnw 'packages/manager/apps/nutanix/src' -e 'moment' | wc -l
0
```

**Review by antleblanc — CHANGES_REQUESTED** [review]
Great job!

Existing a diff on the `yarn.lock` file right after installing this branch.
Parent branch needs to be rebased I think.

Relates to #5429, `@ovh-ux/ui-kit` has been bumped to `v5.1.0`.
https://github.com/ovh/manager/blob/master/yarn.lock#L3013-L3014

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/template.html [hunk 4]
```diff
@@ -19,5 +19,57 @@
...
+        <div class="col-md-4">
+            <oui-tile heading="Support" loading="$ctrl.loading">
+                <oui-tile-definition
+                    term="Support level"
+                    description="standard"
+                >
+                </oui-tile-definition>
+                <oui-tile-definition term="Tickets ouverts">
+                    <oui-tile-description>
+                        <a class="oui-link_icon" data-ng-href="go/somewher">
```
[the remark this answers is not in the document]
Comment (varun257) [reply packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/template.html]: Wrong code...have removed this

**Comment by antleblanc** [comment]
Looks like following commit `843f3d7` has a typo.
Shouldn't be `management` instead of `managemennt`?

**Comment by varun257** [comment]
> Great job!
> 
> Existing a diff on the `yarn.lock` file right after installing this branch.
> Parent branch needs to be rebased I think.
> 
> Relates to #5429, `@ovh-ux/ui-kit` has been bumped to `v5.1.0`.
> https://github.com/ovh/manager/blob/master/yarn.lock#L3013-L3014

Yes we need to rebase the main branch with yarn and "bm-server-components"

File: packages/manager/apps/nutanix/src/app.module.js [hunk 5]
```diff
@@ -48,6 +49,21 @@ export default (containerEl, environment) => {
...
+      /* @ngInject */ ($translate) => {
+        let lang = $translate.use();
+
+        if (['en_GB', 'es_US', 'fr_CA'].includes(lang)) {
+          lang = lang.toLowerCase().replace('_', '-');
+        } else {
+          [lang] = lang.split('_');
+        }
+
+        return import(`script-loader!moment/locale/${lang}.js`).then(() =>
```
Comment (antleblanc) [inline packages/manager/apps/nutanix/src/app.module.js:62]: webpack configuration should apply following plugin `ContextReplacementPlugin`.
see (as an example): 
https://github.com/ovh/manager/blob/361d0dfaba9d3d8bb531f2765a038b866cbe5571/packages/manager/apps/dedicated/webpack.config.js#L108-L111

**Review by antleblanc — CHANGES_REQUESTED** [review]
Some checks were not successful.
Could you please have a look?
Thank you!

File: packages/manager/apps/nutanix/src/app.module.js (hunk 5 above)
Comment (varun257) [reply packages/manager/apps/nutanix/src/app.module.js:62]: Yes in dedicated, but in standalone `nutanix` module, we have to add this.

File: packages/manager/modules/nutanix/src/dashboard/general-info/routing.js [hunk 6]
```diff
@@ -3,6 +3,8 @@ export default /* @ngInject */ ($stateProvider) => {
     url: '',
     component: 'nutanixGeneralInfo',
     resolve: {
+      serviceInfo: /* @ngInject */ (NutanixDashboard, serviceName) =>
+        NutanixDashboard.getServiceInfo(serviceName),
```
Comment (radireddy) [inline packages/manager/modules/nutanix/src/dashboard/general-info/routing.js:7]: Do you think NutanixDashboard should be renamed to "NutanixService" or just "Nutanix". I dont thik we need to have a separate service only for dashboard.

Comment (varun257) [reply packages/manager/modules/nutanix/src/dashboard/general-info/routing.js:7]: I think it makes sense to make it as `NutanixService`

**Review by radireddy — APPROVED** [review]

**Review by ganeshkumar1989 — APPROVED** [review]

**Pull request merged**